### DISCUSSION TITLE
Is it possible support zipped sound dir? and how?

### DISCUSSION THREAD

**Discussion by zw963** [goldendict/goldendict#1413] [Q&A] [answered]
Please check following screnshot.

![image](https://user-images.githubusercontent.com/549126/132995562-653720d8-96b0-48df-9cab-6f553e62c053.png)

Dropbox/dict/voices/ folder exists many mp3 files, like this:

```
 ╰─ $ tree -L 1
.
├── A
├── B
├── C
├── D
├── E
├── F
├── G
├── H
├── I
├── J
├── K
├── L
├── M
├── N
├── O
├── P
├── Q
├── R
├── S
├── T
├── U
├── V
├── W
├── X
├── Y
└── Z
```

there exist many mp3 files in each subfolder, is it possible package those mp3 into a zipped file, and still can use it directly on goldendict without need extract it?

Thank you!

**Comment by Abs62** [comment]
Did you read Goldendict help about Zips format?

**Reply by zw963** [reply]
> Did you read Goldendict help about Zips format?

Sorry, i guess not, where to read it?

i only check help  from following position, no see any useful help for current case.

![image](https://user-images.githubusercontent.com/549126/133025489-34788255-9b65-4c0c-b89b-337dcbdfcc7d.png)

**Comment by zw963** [comment]
Hi, i find the document now.

```
7. Zips sound packs
Zips sound pack is zip archive with extension ".zips" contains set of sound files. To create suck pack it needs to compress sound files into zip archive and change extension of this archive to ".zips". GoldenDict will handle such sound packs like ABBYY Lingvo sound archives.
Zips file size must not exceed 4 GB. 
```

But it not work, i download a zipped file from http://www.lingoes.net/en/translator/speech.htm
and, place it into folder named:  ``/home/zw963/Dropbox/dict/voices/'Lingoes English.zips'``, and add it into sound dirs

![image](https://user-images.githubusercontent.com/549126/133026652-c88e4dfa-957d-4b50-ae5a-59ba206617c7.png)

I use mpv as external player. (it works if extracted mp3 files out of zip), but it not work whe use with zips file after i add above voice into current group.

**Reply by nonwill** [reply]
![20210913140258](https://user-images.githubusercontent.com/46510529/133031813-d1910e2b-5dfb-411a-b5d6-f574bfbc1d66.png)
Just add the path to File sources.

**Reply by zw963** [reply]
> ![20210913140258](https://user-images.githubusercontent.com/46510529/133031813-d1910e2b-5dfb-411a-b5d6-f574bfbc1d66.png)
> Just add the path to File sources.

Still not work.

```
 ╭─ 16:07  zw963 ⮀ ~/Dropbox/dict/voices ⮀ ⭠ (acd168992) billy-debug *$% u=  ➦ ruby-3.0.2  0 
 ╰─ $ ls
'Lingoes English.zips'

```

I can see dict was added into dictionary, but still no sound.

![image](https://user-images.githubusercontent.com/549126/133049108-a9ba84ec-0bc7-495b-af32-4b193f818467.png)

After i add LDOE5++ En-Cn V2.15, i can hear the voice again, so, that should not voice config issue anyway.

**Reply by nonwill** [reply]
> I can see dict was added into dictionary, but still no sound.

That's OK. You can now view its headwords and lookup words from it:
![20210913172016](https://user-images.githubusercontent.com/46510529/133058585-78e49886-b57b-4918-87e1-fe26b1a7e94d.png)

**Comment by zw963** [comment]
Wired, but, why voice start to no working after i add another dict into this group.

![image](https://user-images.githubusercontent.com/549126/133098889-d38ff204-9c25-4016-a339-6f9253df59e9.png)

no sound. (it works if use mp3 format with other dictionaries.)

![image](https://user-images.githubusercontent.com/549126/133099028-d3bd399b-4e48-41dc-83ec-c115c1af35d8.png)

**Reply by nonwill** [reply]
Please try H/house.

**Comment by zw963** [comment, marked as answer]
Okay, it works after re compress all mp3 file into a zip which flatten all mp3 to top folder. thank you
